### PR TITLE
Fix metronome not reviving when starting new job

### PR DESCRIPTION
**Pull request by alenkacz**
Summary:
Offer matcher reconciler was not run with `.start()` so the source was emitting no value and `combineLatest` needs at least one value to publish *ANY* result.

We don't need OfferMatcherReconciler at all since we don't use PVs (as pointed out by Matthias).

JIRA issues: DCOS_OSS-5166

### CODE REVIEW

File: jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala [hunk 1]
```diff
@@ -97,9 +97,6 @@ class SchedulerModule(
       persistenceModule.groupRepository,
       leadershipModule)(actorsModule.materializer)
 
-  lazy val taskStatusProcessor: TaskStatusUpdateProcessor = new TaskStatusUpdateProcessorImpl(
```
Comment (alenkacz) [inline jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala]: this is unrelated, it was just unused field

**Comment by timcharper** [comment]
This fixes the issue upstream: https://github.com/mesosphere/marathon/pull/6948

File: jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala [hunk 2]
```diff
@@ -88,22 +88,9 @@ class SchedulerModule(
...
-      instanceTrackerModule.instanceTracker,
-      persistenceModule.groupRepository,
-      leadershipModule)(actorsModule.materializer)
-
-  lazy val taskStatusProcessor: TaskStatusUpdateProcessor = new TaskStatusUpdateProcessorImpl(
-    metrics, clock, instanceTrackerModule.instanceTracker, schedulerDriverHolder, killService, eventBus)
-
   private[this] lazy val launcherModule: LauncherModule = {
     val instanceTracker: InstanceTracker = instanceTrackerModule.instanceTracker
     val offerMatcher: OfferMatcher = StopOnFirstMatchingOfferMatcher(
```
Comment (meichstedt) [inline jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala]: Take a look at `StopOnFirstMatchingOfferMatcher`: you don't even need to use that one and could just do
```
val offerMatcher: OfferMatcher = offerMatcherManagerModule.globalOfferMatcher
```

:D

Comment (alenkacz) [reply jobs/src/main/scala/dcos/metronome/scheduler/SchedulerModule.scala]: LOL :) You're right as always :)

**Comment by kensipe** [comment]
ugh
```
+ bin/install-protobuf.sh
Linux
bin/install-protobuf.sh: 10: [: Linux: unexpected operator
E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)
E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?
script returned exit code 100
```
